google/ads/google_ads/v2/services/enums.py

```diff
@@ -5444,19 +5373,19 @@ class ExternalConversionSource(enum.IntEnum):
 class FeedAttributeOperation(_CreateEnumTypeUponFirstAccess):
     Operator = '''\
     class Operator(enum.IntEnum):
-    """
-    The operator.
+        """
+        The operator.
 
-    Attributes:
-      UNSPECIFIED (int): Unspecified.
-      UNKNOWN (int): Used for return value only. Represents value unknown in this version.
-      ADD (int): Add the attribute to the existing attributes.
-    """
-    UNSPECIFIED = 0
-    UNKNOWN = 1
-    ADD = 2
+        Attributes:
+          UNSPECIFIED (int): Unspecified.
+          UNKNOWN (int): Used for return value only. Represents value unknown in this version.
+          ADD (int): Add the attribute to the existing attributes.
+        """
+        UNSPECIFIED = 0
+        UNKNOWN = 1
+        ADD = 2
 '''
-FeedAttributeOperation = FeedAttributeOperation()
+FeedAttributeOperation = FeedAttributeOperation() # For __getattribute__
```

Comment:
could you either expand or remove this comment?

Reply:
This is just for consistency, since it's next to all of the other corresponding lines in this file.